docs/connectors/io-crd-config/sink-crd-config.md

```diff
@@ -171,3 +171,4 @@ Function Mesh supports customizing the Pod running Pulsar connectors. This table
 | `builtinAutoscaler` | Specify the built-in autoscaling rules. <br /> - CPU-based autoscaling: auto-scale the number of Pods based on the CPU usage (80%, 50%, or 20%). <br />- Memory-based autoscaling: auto-scale the number of Pods based on the memory usage (80%, 50%, or 20%). <br /> If you configure the `builtinAutoscaler` field, you do not need to configure the `autoScalingMetrics` and `autoScalingBehavior` options and vice versa.|
 | `autoScalingMetrics` | Specify how to scale based on customized metrics defined in connectors. For details, see [MetricSpec v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#metricspec-v2beta2-autoscaling). |
 | `autoScalingBehavior` | Configure the scaling behavior of the target in both up and down directions (`scaleUp` and `scaleDown` fields respectively). If not specified, the default Kubernetes scaling behaviors are adopted. For details, see [HorizontalPodAutoscalerBehavior v2beta2 autoscaling](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#horizontalpodautoscalerbehavior-v2beta2-autoscaling). |
+| `env` | Specify the environment variables to expose on the containers. It is a key/value map. You can either specify a particular value for the environment variable or use the `valueFrom` option to specify the [source](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#envvarsource-v1-core) for the environment variable's value. |
```

Comment:
Now sure if this is clear.

It is a list like:

```
- name: "EXAMPLE_1"
   value: "VALUE_1"
- name: "EXAMPLE_2"
  value: "VALUE_2" 
 ```

Reply:
@jdmaguire Thanks for your comment. I add a YAML code example in the table cell. It looks like this. Could you please help whether it is OK or not? Thanks.
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/48120384/181705320-99afb3f9-dddb-4c6d-bb04-86dccfccde50.png">